substrate/bin/node/executor/src/lib.rs

```diff
@@ -15,26 +15,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! A `CodeExecutor` specialization which uses natively compiled runtime when the wasm to be
-//! executed is equivalent to the natively compiled code.
+//! Provides executor related types intended to use accross substrate node.
 
-pub use sc_executor::NativeElseWasmExecutor;
+/// Host functions required for kitchensink runtime and substrate node.
+pub type HostFunctions =
```

Comment:
Iirc these should be extensions to the default (I.e. SubstrateHostFunctions are not explicitly required here)?